bodhi/server/util.py

```diff
@@ -343,6 +343,20 @@ def status2html(context, status):
     return "<span class='label label-%s'>%s</span>" % (cls, status)
 
 
+def ci_status2html(context, status):
+    cls = {
+        'ignored': 'success',
+        'running': 'warning',
+        'passed': 'success',
+        'failed': 'danger',
+        'queued': 'info',
+        'unknown': 'primary',
+    }[status]
+    if status == 'unknown':
+        status = 'info awaited'
```

Comment:
I propose another way: ``op.execute("UPDATE builds set ci_status='ignored'")`` in the alembic migration.

Not displaying the CI status sounds more like something we would want to condition based on a configuration key.